pyaerocom/aeroval/fairmode_statistics.py

```diff
@@ -118,6 +153,28 @@ def _get_stats(
 
         return stats_list
 
+    @staticmethod
+    def _exceedances(data: xr.DataArray, var_name: str) -> list[np.array]:
+        if var_name == "concno2":
+            new_data = data.resample(time="1D", skipna=True).max()
+        else:
+            new_data = data
+
+        obsvals = new_data.data[0]
+        modvals = new_data.data[1]
+
+        mask = np.isfinite(obsvals) & np.isfinite(modvals)
+        obsex = np.sum(obsvals > EXC_THRESHOLDS[var_name], axis=0, where=mask)
+        modex = np.sum(modvals > EXC_THRESHOLDS[var_name], axis=0, where=mask)
+
+        return [obsex, modex]
+
+    @staticmethod
+    def _NMB(x: np.ndarray, y: np.ndarray) -> np.ndarray:
+        num = np.nansum(x - y, axis=0)
+        denum = np.nansum(x, axis=0)
+        return np.where(denum == 0, np.nan, num / denum)
+
     @staticmethod
     def pearson_R(x: np.ndarray, y: np.ndarray) -> np.ndarray:
         mask = ~np.isnan(x) * ~np.isnan(y)
```

Comment:
I know, this line has been there for a while, but it is the same as we fixed other places...